src/components/Register/AdditionalDetailsSection.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="additional-details-section__title">
+    <SectionTitle variation="40-60-80">Great!</SectionTitle>
+    <SectionTitle color="bombay" variation="40-60-80"
+      >Now your name</SectionTitle
+    >
+  </div>
+  <FormCard @submit="(event) => handleSubmit(event)">
```

Comment:
Also FYI - this is the same as the following:
```suggestion
  <FormCard @submit="handleSubmit">
```